docs/install/index.md

```diff
@@ -689,11 +689,50 @@ $ bash install-mxnet-osx-python.sh
   <div class="python">
     <div class="gpu">
 
-More details and verified installation instructions for macOS, with GPUs, coming soon.
+<div class="pip virtualenv docker">
+</br>
+
+Try the **Build from Source** option for now.
+
+</div>
+
+<div class="build-from-source">
+
+The following instructions are for CUDA 9.1 and cuDNN 7 for MacOS X 10.12+ and a CUDA-capable GPU. They summarize confirmed successful builds in [#9217](https://github.com/apache/incubator-mxnet/issues/9217).
+Alternatively, you may follow the [CUDA installation instructions for Mac OS X](https://docs.nvidia.com/cuda/cuda-installation-guide-mac-os-x/index.html).
+
+1. [Download Xcode 8.3.3 from Apple](https://download.developer.apple.com/Developer_Tools/Xcode_8.3.3/Xcode8.3.3.xip). This is the version [NVIDIA specifies in its instructions for Mac OS X](https://docs.nvidia.com/cuda/cuda-installation-guide-mac-os-x/index.html). Unzip and rename to `Xcode8.3.3.app`.
+
+2. Run `sudo xcode-select -s /Applications/Xcode8.3.3.app` or to wherever you have placed Xcode.
+
+3. Run `xcode-select --install` to install all command line tools, compilers, etc.
+
+4. Install CUDA for MacOS X. Specific steps are provided in NVIDIA's [CUDA installation instructions](https://docs.nvidia.com/cuda/cuda-installation-guide-mac-os-x/index.html#installation).
+
+5. Run `git clone --recursive https://github.com/apache/incubator-mxnet.git mxnet` to get the latest version.
+
+6. Run `cd mxnet`.
 
+7. Edit the `make/osx.mk` file to set the following parameters:
 
-*MXNet* is expected to be compatible on macOS with NVIDIA GPUs. Please install CUDA 9.0 and cuDNN 7, prior to installing GPU version of *MXNet*.
+    ```
+    USE_CUDA = 1
+    USE_CUDA_PATH = /usr/local/cuda
+    USE_CUDNN = 1
+    USE_OPENCV = 0
```

Comment:
It might be better to have this conversation along with the source of these instructions: #9217 
I was mostly trying to get the latest validated instructions presented and have the install selector actually work.
Also, I purposely avoided OpenCV because it is a can of worms and instructions depend on the version you want, so, overall, it can be a royal pain to get working, whereas a GPU build without OpenCV is much more easily attained. 
Happy to add it if there is a **validated** build chain with specific versions of it mentioned. I'm afraid just switching to `use_opencv=1` may break these instructions.
Regarding the custom script - again, that needs to be verified, and the old getting started seems pretty out of date... whereas this recent issue seems to be more current.
I think we should make a note of it and provide the brew instructions, but let this pass for now until we have validated instructions for opencv or a custom install script.